### ISSUE TITLE
 Reference links about MAAS

### ISSUE THREAD

**Issue by Cwpute** [vbresan/MindfulAttentionAwarenessScale#10]
In the about section of your app you talk about studies and experiments about the MAAS, but you don't actually provide links to any of them.
Please do so that we are able to browse them ourselves.

**Comment by vbresan** [comment]
Awesome! Feel free to proceed with the issue.

**Comment by Cwpute** [comment]
i'm not knowledgeable about MAAs at all, and i don't have the time to research it either… i figured you could provide these studies yourself.

**Comment by vbresan** [comment]
In other words, you would prefer that someone else googles it for you?

**Comment by Cwpute** [comment]
If getting reliable, scientific information was only a google search away, we wouldn't live in such a twisted world ;)
If you don't want to or are unable to provide it yourself, that's fine, someone else could.

**Comment by vbresan** [comment]
Yes, scientific research papers are just a Google search away. Apparently, we live in a twisted world for other reasons. Instead of making excuses, you could have already found what you claim to be looking for.

**Comment by vbresan** [comment]
You opened the issue requesting a link to a study that's mentioned in the About section — a link that's just a quick search away. Now, you're excusing yourself by saying you don't have time to research the whole topic of mindfulness. By all means, you are at the wrong place.

**Comment by Cwpute** [comment]
The "About the score" section only mentions "studies", without any other information about either who directed them or where they where published. This is not enough to search for them, and deduct which ones exactly this section is refering to. Am i missing something ?

**Comment by vbresan** [comment]
Yes, you missed reading the app's 'About' section before deciding to install it, and you also missed reading the 'About' section of the repository README file before deciding to open issues.

**Comment by vbresan** [comment]
I certainly hope that other users do pay more attention. The web page that you shared includes a direct link to the full paper. Now it's time that you modify the README and make the paper's title link directly to the PDF.

**Comment by Cwpute** [comment]
I don't know how to do all that, and as i said i won't invest the time for it someone else will have to.
Thanks for your patience.

Le 17 décembre 2024 10:07:23 GMT+01:00, "Viktor Brešan" ***@***.***> a écrit :

**Comment by vbresan** [comment]
It's not difficult, and it won’t take more time than you've already invested. It's an opportunity to learn something new and grow. Plus, you'll immortalize yourself by becoming a contributor to this project.

**Comment by Cwpute** [comment]
I will say it one last time: i don't and i won't make the time for it.
Several times you had the opportunity to get us both out of this unpleasant situation. I was ready to leave it at my previous message for an information that i had missed, my apologies, but you chose to drag it further on. You've been anything but helpful and courteous in your answers so far, and your late patronizing tone won't incite me into any further conversation or contributions.

**Comment by vbresan** [comment]
Just as I thought, you came back to leave another comment instead of using the same amount of time in a constructive way. And here you go again:

**Comment by Cwpute** [comment]
Yeah okay right, good luck with your "mindfulness" then :D
